codecov.yml

```diff
@@ -1,13 +1,18 @@
+codecov:
+  branch: develop
```

Comment:
I think this is unnecessary based [on this](https://docs.codecov.com/docs/codecov-yaml#default-branch):

"Codecov will use the default branch from Git for your repository as the primary source of the repositories' Codecov.yml."

ie if the default branch ever changes in git, this would prevent codecov from moving to use that new default branch is how I read it - does that seem right to you?

Reply:
ok gotcha - that makes sense, thx 👍